client/rest/src/main/java/org/elasticsearch/client/ResponseException.java

```diff
@@ -30,7 +30,7 @@
  * Exception thrown when an elasticsearch node responds to a request with a status code that indicates an error.
  * Holds the response that was returned.
  */
-public final class ResponseException extends IOException {
+public class ResponseException extends IOException {
```

Comment:
I think that these changes to ResponseException are no longer needed?

Reply:
doh, good call!